src/main/java/pl/oskarkowalski/recruitment/ticketbookingapp/enums/TicketPrice.java

```diff
@@ -0,0 +1,31 @@
+package pl.oskarkowalski.recruitment.ticketbookingapp.enums;
+
+import lombok.Getter;
+
+@Getter
+public enum TicketPrice {
+  ADULT((double) 25),
+  STUDENT((double) 18),
+  CHILD(12.5),
+  ADULT_WEEKEND((double) 29),
+  STUDENT_WEEKEND((double) 22),
+  CHILD_WEEKEND(16.5);
+
+  private final Double amount;
```

Comment:
dlaczego duży Double?

Reply:
Zastosowanie dużego Double'a tutaj jest błędem, wystarczałby prymityw zamiast wrappera, ale to i tak jest docelowo do zastąpienia przez BigDecimal.